renderer/components/nettests/websites/WebConnectivity.js

```diff
@@ -29,32 +29,30 @@ const WebConnectivity = ({measurement, isAnomaly, render}) => {
   }
 
   const WebDetails = () => (
-    <FullHeightFlex>
-      <Box>
-        <Flex my={3} flexDirection='column'>
-          {isAnomaly ? (
-            <Box>
-              <FormattedMarkdownMessage
-                id='TestResults.Details.Websites.LikelyBlocked.Content.Paragraph'
-                values={{
-                  WebsiteURL: url,
-                  BlockingReason: blockingReason
-                }}
-              />
-            </Box>
-          ) : (
-            <Box>
-              <FormattedMessage
-                id='TestResults.Details.Websites.Reachable.Content.Paragraph'
-                values={{
-                  WebsiteURL: url
-                }}
-              />
-            </Box>
-          )}
-        </Flex>
-      </Box>
-    </FullHeightFlex>
+    <Box>
+      <Flex my={3} flexDirection='column'>
+        {isAnomaly ? (
+          <Box>
+            <FormattedMarkdownMessage
+              id='TestResults.Details.Websites.LikelyBlocked.Content.Paragraph'
+              values={{
+                WebsiteURL: url,
+                BlockingReason: blockingReason
+              }}
+            />
+          </Box>
+        ) : (
+          <Box>
+            <FormattedMessage
+              id='TestResults.Details.Websites.Reachable.Content.Paragraph'
+              values={{
+                WebsiteURL: url
+              }}
+            />
+          </Box>
+        )}
+      </Flex>
+    </Box>
```

Comment:
You moved the `FullHeightFlex` up one level (just to check I'm understanding the PR flow)

Reply:
Exactly. Inspired by you, I started using more detailed commit messages like in this case a28888cb4a1ce23f460185ac5efb379c67386a1f.